_data/language.yml

```diff
@@ -1,6 +1,7 @@
 ---
 footer:
-  About the Commission: The CSIS Commission on Strengthening America’s Health Security
+  about_the_commission_heading: About the Commission
```

Comment:
You can rename these to `about_heading` and `about_desc` respectively. Since they're contained within the `footer` section, there's no risk of overlap for other language.